tests/tap.spec.ts

```diff
@@ -58,7 +58,7 @@ it('trial run should not tap', async ({}) => {
   await page.tap('#a');
   const eventsHandle = await trackEvents(await page.$('#b'));
   await page.tap('#b', { trial: true });
-  expect(await eventsHandle.jsonValue()).toEqual([]);
+  expect(await eventsHandle.jsonValue()).toEqual(['pointerover', 'pointerenter', 'pointerout', 'pointerleave']);
```

Comment:
I don't like the idea that the trail run has user visible events, but at the very least this test needs to be rewritten so that it actually tests something. In its current form it would pass with `{ trial: false }`.

Reply:
Not really, with `{trial:false}` we'll get a lot more events - that's what the previous test does.